runjags and rjags installation

On Travis-CI, e.g., https://travis-ci.org/USGS-R/streamMetabolizer/builds/70088636#L1478:
```
* installing *source* package ‘runjags’ ...
** package ‘runjags’ successfully unpacked and MD5 sums checked
checking for prefix by checking for jags... no
configure: error: "Location of JAGS headers not defined. Use configure arg '--with-jags-include' or environment variable 'JAGS_INCLUDE'"
ERROR: configuration failed for package ‘runjags’
* removing ‘/usr/local/lib/R/site-library/runjags’
```
saw similar issues with rjags on Condor before I [temporarily] removed it.

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,7 +6,7 @@ r_github_packages:
  - jimhester/covr
  - appling/unitted
 
- install:
+install:
   - wget http://downloads.sourceforge.net/project/mcmc-jags/JAGS/3.x/Source/JAGS-3.4.0.tar.gz
   - tar xfvs JAGS-3.4.0.tar.gz
   - cd JAGS-3.4.0

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,6 +6,15 @@ r_github_packages:
  - jimhester/covr
  - appling/unitted
 
+ install:
+  - wget http://downloads.sourceforge.net/project/mcmc-jags/JAGS/3.x/Source/JAGS-3.4.0.tar.gz
+  - tar xfvs JAGS-3.4.0.tar.gz
+  - cd JAGS-3.4.0
+  - sudo ./configure
+  - sudo make
+  - sudo make install
+  - cd ..
+  
 after_success:
   - Rscript -e 'covr::coveralls()'
 

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,7 +6,7 @@ r_github_packages:
  - jimhester/covr
  - appling/unitted
 
-install:
+before_install:
   - wget http://downloads.sourceforge.net/project/mcmc-jags/JAGS/3.x/Source/JAGS-3.4.0.tar.gz
   - tar xfvs JAGS-3.4.0.tar.gz
   - cd JAGS-3.4.0

